cypress/integration/Popover/position/index.js

```diff
@@ -48,7 +48,7 @@ Given(
 Given(
     'there is sufficient space between the bottom of the reference element and the bottom of the Popover to show the arrow',
     () => {
-        getRefAndPopoverPositions().then(([refPos, popoverPos]) => {
+        getRefAndPopoverPositions().should(([refPos, popoverPos]) => {
```

Comment:
This relates to my previous comment. Here we are now using the `getRefAndPopoverPositions` helper, which just calls `cy.getPositionsBySelectors`. So now we have three flavours of doing these position assertions:
- Using a named helper that calls `cy.getPositionsBySelectors`
- Using `cy.getPositionsBySelectors` directly (I think this is probably the way to go)
- Not using a helper, as in https://github.com/dhis2/ui/commit/f355ea6889e7217f91bdbf7cfd932be2bc0063a8

So it seems we are introducing inconsistency here. It's not a big deal, but I think my personal preference would be that we either:
1. Consistently call `cy.getPositionsBySelectors` directly in this PR, and update the position assertions for the Select too.
2. Focus purely on refactoring `.then()` to `.should()`, and don't touch the position assertions themselves. This would mean reverting some of the changes in this PR too (i.e.`cypress/integration/MenuItem/position/index.js`).

Reply:
See https://github.com/dhis2/ui/pull/102#discussion_r429929966